src/Features/Core/Portable/Diagnostics/Analyzers/PreferFrameworkTypeDiagnosticAnalyzerBase.cs

```diff
@@ -31,16 +31,7 @@ private static PerLanguageOption<CodeStyleOption<bool>> GetOptionForDeclarationC
         private static PerLanguageOption<CodeStyleOption<bool>> GetOptionForMemberAccessContext 
             => CodeStyleOptions.PreferIntrinsicPredefinedTypeKeywordInMemberAccess;
 
-        public override bool OpenFileOnly(Workspace workspace)
-        {
-            var preferTypeKeywordInDeclarationOption = workspace.Options.GetOption(
```

Comment:
same question. do we know why it was open file only before?

Reply:
I think OpenFileOnly is that special IBuiltInAnalyzer stuff.  Do we know who added that, so we can find out?

Reply:
I added it due to IDE analyzer special behavior I listed above and below. but that doesn't mean I know why these analyzers are chosen to be open file only.

Reply:
➡️ The changes to `OpenFileOnly` methods were reverted (no other changes so I amended the previous commit)